docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/DnnFeaturizeImage.cs

```diff
@@ -15,7 +16,7 @@ public static void Example()
             // Downloading a few images, and an images.tsv file, which contains a list of the files from the dotnet/machinelearning/test/data/images/.
             // If you inspect the fileSystem, after running this line, an "images" folder will be created, containing 4 images, and a .tsv file
             // enumerating the images. 
-            var imagesDataFile = SamplesUtils.DatasetUtils.DownloadImages();
+            var imagesDataFile = Microsoft.ML.SamplesUtils.DatasetUtils.DownloadImages();
```

Comment:
>Microsoft.ML. [](start = 33, length = 13)

Do you need this?

Reply:
added to `ConvertToGrayScale` in iteration 2  (else its compile time error)

---
In reply to: [272409998](https://github.com/dotnet/machinelearning/pull/3208#discussion_r272409998) [](ancestors = 272409998)